ha/setup/create_pacemaker_resources.py

```diff
@@ -246,12 +264,38 @@ def config_constraint(cib_xml, push=False, **kwargs):
     if push:
         cib_push(cib_xml)
 
+
+def configure_stonith(cib_xml, push=False, **kwargs):
+    """Create ipmi stonith resource."""
+    try:
+        stonith_resource_id = str(kwargs["stonith_resource_id"])
```

Comment:
We need to pass these parameters from the caller (ha_setup.py). Please make sure we are doing that. Are these values present in the current provisioner confstore file?
Separately, how the stonith configuration will happen for all nodes in the cluster since I think the create_resource() will be called from one node in the cluster only.

Reply:
@indrajitzagade I have updated the code to support stonith config on multiple nodes.

Reply:
I think with the updated code still stonith will not be configured for all nodes in the cluster, it will be configured just on first node.

Reply:
Done code updated and tested with config command with ha_setup.